src/components/Tabs/index.tsx

```diff
@@ -4,7 +4,7 @@ import styled, { css } from 'styled-components'
 export const Tabs = styled(TabsRadix.Root)`
   display: flex;
   flex-direction: column;
-  gap: 50px;
+  gap: 57px;
```

Comment:
57px? Trabalha sempre com múltiplo de 8. Sempre números pares de preferência, fica mais harmonico. 1 * 8, 1.5 * 8, 2 * 8, 4 * 8 etc. Ou seja, usa 56 ao invés de 57px. Inclusive, reve tudo ai no projeto pra seguir esse padrão. Pode até criar uma função spacing, que recebe um valor e multiplica por 8. Por exemplo:
```javascript
padding: ${spacing(1.0)};
```

Reply:
What you think about this [way](https://github.com/joaooliveiradev/conduit-frontend/pull/39/commits/37f90a771cbb8af8a8d24d1e9a8a2e81f439b666)

Some spacings i needed to put the closest multiple of eight because they were not a multiple

Reply:
Because i needed to put the closest, some spacings changed and they are not the same as the figma design, what you think about this?